client/my-sites/site-settings/style.scss

```diff
@@ -625,3 +625,38 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 	margin-top: -16px;
 	margin-bottom: 16px;
 }
+
+.site-settings__advanced-customization-notice {
+	display: flex;
+	padding: 20px;
+	background-color: #f4f4f4;
+	align-items: center;
+	justify-content: space-between;
+
+	.site-settings__advanced-customization-notice-cta {
+		display: flex;
+		align-items: center;
+	}
+
+	svg.gridicon.gridicons-info-outline {
+		color: #d38d16;
+		padding-right: 10px;
+	}
+
+	.site-settings__advanced-customization-notice-buttons {
+		display: flex;
+
+		.button.is-primary {
+			margin-left: 10px;
+
+		}
+	}
+
+	@media (max-width: 1100px) {
```

Comment:
Any reason why this is not using any of the existing breakpoints? See https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries or https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/stylesheets/shared/mixins/_breakpoints.scss

Reply:
Changed, thanks for pointing to the file!